cmake/developer_package/version.cmake

```diff
@@ -94,20 +94,27 @@ macro(ie_parse_ci_build_number)
     message(STATUS "OpenVINO version is ${IE_VERSION}")
 endmacro()
 
-if (DEFINED ENV{CI_BUILD_NUMBER})
-    set(CI_BUILD_NUMBER $ENV{CI_BUILD_NUMBER})
-else()
-    branchName(GIT_BRANCH)
-    commitHash(GIT_COMMIT_HASH)
-
-    set(custom_build "custom_${GIT_BRANCH}_${GIT_COMMIT_HASH}")
-    set(CI_BUILD_NUMBER "${custom_build}")
-endif()
-
-# provides Inference Engine version
-# 1. If CI_BUILD_NUMBER is defined, parses this information
-# 2. Otherwise, parses ie_version.hpp
+macro (ie_set_version IE_VERSION)
```

Comment:
Why do we need this change?

Reply:
This is required for applications like AIRA to function correctly as they require OpenVINO version as the first part of string in format above, see EISW-41342 for details.